back/app/services/input_ui_schema_generator_service.rb

```diff
@@ -56,6 +56,7 @@ def default_options(field)
       defaults[:answer_visible_to] = field.answer_visible_to
     end
     defaults[:otherField] = field.other_option_text_field&.key if field.other_option_text_field
+    defaults[:dropdown_layout] = field.dropdown_layout if %w[multiselect select].include?(field.input_type)
```

Comment:
If we go for my suggestion in [this comment](https://github.com/CitizenLabDotCo/citizenlab/pull/8625/files#r1711499440), then we could use...

```Ruby
defaults[:dropdown_layout] = field.dropdown_layout if field.dropdown_layout_type?
```